Jenkinsfile

```diff
@@ -44,14 +42,8 @@ pipeline {
             )
           }
 
-          // Always run the full pipeline on nightly builds
-          expression { params.NIGHTLY }
-
-          // Always run the full pipeline when triggered by conjur build
-          expression { getTrigger() == "upstreambuild" }
-
-          // Always run the full pipeline on tags of the form v*
-          tag "v*"
```

Comment:
we run tag pipelines now too - you may consider moving this project to tag-triggered builds. the benefit is that you run `bin/publish` on the condition `tag "v*"` instead of using a condition like this:
https://github.com/cyberark/secrets-provider-for-k8s/blob/5c2a51375eb6c1a39f91ee7169feaca7c9e4f30a/bin/publish#L22

the problem with your current publishing strategy is that if a new commit is published to master before the tag is pushed, your master build may not meet this condition and your tagged artifacts may not be published. using the tag-triggered pipelines gets around this risk, but it will mean you need to add back this condition to the Jenkinsfile.